server/mediaserver.go

```diff
@@ -1308,7 +1309,7 @@ func (s *LivepeerServer) HandleRecordings(w http.ResponseWriter, r *http.Request
 			w.WriteHeader(http.StatusNotFound)
 			return
 		}
-		s.streamMP4(w, r, mainJspl, manifestID, track)
+		s.streamMP4(w, r, mainJspl, manifestID, track, pp[len(pp)-1])
```

Comment:
Cool - this means that `source.mp4` as well as `source/streamName.mp4` will both work, yeah? Sounds great.

Reply:
Yes